flag.go

```diff
@@ -10,6 +11,10 @@ import (
 	"time"
 )
 
+var (
+	slPfx = fmt.Sprintf("sl:::%d:::", time.Now().UTC().UnixNano())
```

Comment:
This is me being paranoid.  I'm trying to minimize the risk that we'd deserialize something coming out of the stdlib `flag` layer that's not "ours" to deserialize.  My concern isn't so much about safety, but rather aiming for clarity, avoiding unexpected bugs, making it super easy to spot when a string value from the cli layer isn't being handled correctly.